cmd/renterd/node.go

```diff
@@ -328,6 +320,77 @@ func newBus(cfg config.Config, pk types.PrivateKey, network *consensus.Network,
 		}
 	}
 
+	// create chain database
+	chainPath := filepath.Join(cfg.Directory, "consensus.db")
+	consensusExists := fileExists(chainPath)
+
+	var bdb *coreutils.BoltChainDB
+	var store *chain.DBStore
+	var state consensus.State
+	if !cfg.Explorer.Disable && instantSync && !consensusExists {
+		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+		defer cancel()
+
+		walletAddress := types.StandardUnlockHash(pk.PublicKey())
+		exp := explorer.NewExplorer(cfg.Explorer.URL)
```

Comment:
This is a bit annoying to use with zen right now. `cfg.Explorer.URL` defaults to `api.siascan.com`.
So if you set your network to `zen`, we will attempt to use a mainnet checkpoint on the zen network which obviously fails.

```
2025-12-03T15:12:42+01:00	INFO	renterd	{"version": "?", "network": "zen", "commit": "?", "buildDate": "1970-01-01T01:00:00+01:00", "config": "renterd.yml"}
2025-12-03T15:12:42+01:00	INFO	sql	Using SQLite version 3.50.4
2025-12-03T15:12:42+01:00	INFO	instant syncing to checkpoint	{"checkpoint": "499665::bbe5502c"}
failed to create node: failed to retrieve checkpoint: failed to retrieve checkpoint from any peer
```